### ISSUE TITLE
[Bug]: Time torch does nothing

### ISSUE THREAD

**Issue by Kilazur** [Mrbysco/ForceCraft#73]
### General Info

- [X] I am running a modpack
- [X] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the mod

### Forge version

43.2.3

### Minecraft version

1.19

### Mod version

3.2.0

### Java version

don't know how to check

### Issue Description

Placing time torches next to processing blocks doesn't increase their speed/reduce their ticks' duration at all.

Basically, nothing happens.

Tested on a server and in a solo world, running the Direwolf20 1.19 modpack.

Tested with Forcecraft's Force Furnace and a regular Minecraft furnace in solo, and multiple block on the server (redstone furnace, bee's nest, Vitalize (mod) setup, and others I don't recall).

**Comment by Mrbysco** [comment]
That's because it wasn't implemented yet on 1.19

**Comment by Kilazur** [comment]
Updated to the latest version after #74 ; torches don't do anything as far as I can tell. Tested with a regular furnace, a Force furnace, and just around growing wheat crops.

**Comment by Mrbysco** [comment]
Are the things directly next to the torch? as it only has a 1 block radius

**Comment by Kilazur** [comment]
Yes
